server/service/get_subscription.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/mattermost/mattermost-plugin-confluence/server/serializer"
 )
 
-const generalError = "Some error occurred. Please try again after sometime."
+const generalError = "some error occurred. Please try again after sometime"
```

Comment:
Just out of curiosity, why was this changed to lowercase? Same with other similar string changes

Reply:
What about the grammar suggestions regarding capitalization? Is there a reason why this was made lowercase?

Reply:
It was a linting error, so I changed it

Reply:
What was the linting error? It seems incorrect to me to display an error message that begins with a lowercase letter, when it is meant to be a sentence. Definitely something we want to fix in our lint settings rather than doing it this way 2/5